recipes/dask-geopandas/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.1.0a2" %}
+
+package:
+  name: dask-geopandas
+  version: {{ version }}
+
+source:
+  url: https://github.com/geopandas/dask-geopandas/releases/download/v{{ version }}/dask-geopandas-{{ version }}.tar.gz
+  sha256: c246ee40ed68e6772ed4e16551c09ddd262ef30b07a8b9ae90686e6ebf94fa34
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - geopandas >=0.9
+    - dask >=2.2.0
+    - distributed >=2.2.0
+
+test:
+  imports:
+    - dask_geopandas
+
+about:
+  home: https://github.com/geopandas/dask-geopandas
```

Comment:
Normally this refers to the `url` in `setup.py`, but interestingly that page doesn't seem to exist atm

https://github.com/geopandas/dask-geopandas/issues/41

Reply:
I opened a PR on the dask-geopandas side to fix this

Reply:
Thanks Joris! I don't think we need to wait for that. Let's resolve that issue asynchronously